runtime/integration-tests/src/generic/utils/democracy.rs

```diff
@@ -1,276 +0,0 @@
-use std::ops::Add;
```

Comment:
I've removed this file because:
- No longer used in our gateway tests, which were correctly covered by its UTs.
- It tests just democracy stuff which is not under our umbrella. We do not want to maintain this.
- No longer used after OpenGov (?): https://github.com/centrifuge/centrifuge-chain/pull/1828

Anyways, I am open to discussing this.

Reply:
I agree with you!